src/main/scala/org/allenai/plugins/archetypes/WebappPlugin.scala

```diff
@@ -25,6 +25,8 @@ object WebappPlugin extends AutoPlugin {
     NodeKeys.nodeProjectDir in Npm := (baseDirectory in thisProject).value / "webapp",
     // Run "npm watch" when we run a re-start.
     Revolver.reStart <<= Revolver.reStart.dependsOn(NodeKeys.nwatch in Npm),
+    // Ensure the node target directory is created before the application tries to start up.
+    Revolver.reStart <<= Revolver.reStart.dependsOn(NodeKeys.mkNodeTarget in Npm),
```

Comment:
Order doesn't matter, but it's true that this should probably be a dependency on `nwatch` instead. I'll fix it!